app/stac_api/models.py

```diff
@@ -361,27 +806,94 @@ def save(self, *args, **kwargs):  # pylint: disable=signature-differs
         # This is needed because save() is called during the Item.object.create() function without
         # calling clean() ! and our validation is done within clean() method.
         self.validate_datetime_properties()
+
         if self.properties_datetime is not None:
+            if self.__original_properties_datetime is not None:
+                # This is the case, when the value of properties.datetime has been
+                # updated
+                self.collection.update_temporal_extent(
+                    'up',
```

Comment:
What do you mean with "up" ? should we not know within the `item.save()` if this is an `insert` or `update`

Reply:
You are right. I implemented this in analogy to the updating of the spatial extent (I think, at least at some state it was the same there, too). But obviously spatial extent has evolved in the meantime and I also updated the logic for differentiating between `insert` and `update` now.